apps/nextra/pages/en/build/cli/formatting-move-contracts.mdx

```diff
@@ -0,0 +1,75 @@
+
+# Formatting Move Contracts
+
+`movefmt` is a formatter tool that makes Move code much easier to write, read, and maintain — greatly improving the development experience on Aptos.
+
+## Installation
+
+`movefmt` is integrated into the Aptos CLI. To begin using it, first install it using the CLI update command.
+
+```bash filename="Terminal"
+# Install movefmt for first time usage
+aptos update movefmt
+```
+
+When a newer version of `movefmt` is available, use the following command to install it:
+
+```bash filename="Terminal"
+# Install movefmt with the target <VERSION>
+aptos update movefmt --target-version <VERSION>
+```
+
+## Format your code
+
+
+Similar to compilation and testing, you can use the following command to format the Move package:
+
+
+```bash filename="Terminal"
+# Format the Move package
+aptos move fmt
+```
+
+Different ways of emitting the formatting result is supported:
+
+```bash filename="Terminal"
+# Format and overwrite the move file `target.move`
+# This is the default behavior if `--emit-mode` is not explicitly specified
+aptos move fmt --emit-mode=overwrite
+
+# Print the formatting result to terminal
+aptos move fmt --emit-mode=std-out
+
+# Print the formatting result to new files
+aptos move fmt --emit-mode=new-file
+
+# Print the difference between before and after formatting
+aptos move fmt --emit-mode=diff
+```
+
+`movefmt` also provides different options to configure how the code will be formatted.
+Here is the default configuration,
+which says 1) each line can have at most 90 characters, 2) the ident is 4 spaces,
```

Comment:
This description might look better as comments right next to the config values, rather than having to manually count and match.